spec/models/game_spec.rb

```diff
@@ -2,21 +2,22 @@
 
 RSpec.describe Game do
   let(:game) { build(:game) }
-  context "When its valid" do
+  context "When it's valid" do
     it "has name" do
-      byebug
       expect(game.name).to be_present
     end
 
-    it { expect(game.game_picture ).not_to be_present }
+    it "has picture" do
+      expect(game.game_picture).to be_present
+    end
   end
-  context "When its invalid" do
+  context "When it's invalid" do
```

Comment:
more like "with invalid attributes" and the other "with valid attributes" this because you don't describe the result on  the text but the conditions that make X result happen. The result is what you write on the expect ( in this case not being valid